src/libstd/sys/unix/thread.rs

```diff
@@ -14,6 +14,7 @@ use prelude::v1::*;
 
 use alloc::boxed::FnBox;
 use cmp;
+#[cfg(not(target_os = "nacl"))]
```

Comment:
This seems to alternate sometimes between a cfg on nacl and on newlib, for example here the import is gated on nacl where the code below is newlib, perhaps these can be consistent throughout?